Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SpacePointOptions.hpp

```diff
@@ -0,0 +1,28 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/TrackFinding/SpacePointMaker.hpp"
+#include "ActsExamples/Utilities/OptionsFwd.hpp"
+
+namespace ActsExamples {
+namespace Options {
+
+/// Add SpacePointMaker options.
+///
+/// @param desc The options description to add options to
+void addSpacePointOptions(Description& desc);
```

Comment:
This naming follows the same pattern as  TrackFindingAlgorithm/TrackFindingOptions in the same directory

Reply:
To me, the name `SpacePointOption` is too general. We can't really tell what is the option used for. Since we don't have to worry about a long naming here, the `SpacePointMakerOption` here and elsewhere sounds more straightforward to the user.

Reply:
@XiaocongAi, I discussed this with @gagnonlg and fixed the name as you suggested.